lib/view_component/slotable_v2.rb

```diff
@@ -140,6 +144,10 @@ def renders_many(slot_name, callable = nil)
           end
         end
 
+        define_method "#{slot_name}?" do
+          send(slot_name).present?
```

Comment:
```suggestion
          get_slot(slot_name).present?
```

same comment as above, re: performance